slopmop/checks/python/lint_format.py

```
@@ -263,6 +376,36 @@ def run(self, project_root: str) -> CheckResult:
+
+    def _check_ruff_imports(self, project_root: str) -> Optional[str]:
+        """Check import order with ruff (equivalent of isort --check-only)."""
+        result = self._run_command(
+            ["ruff", "check", "--select", "I", "."],
+            cwd=project_root,
+            timeout=60,
+        )
+        if not result.success:
+            output = (result.output or "").strip()
+            if output:
+                lines = [l for l in output.splitlines() if l.strip()]
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)